src/tms_service.ts

```diff
@@ -45,6 +52,24 @@ type EmsRasterStyle = {
 };
 
 export class TMSService extends AbstractEmsService {
+  /*
+  List of supported languages with labels and OMT code
+  */
+  public static SupportedLanguages = [
+    { key: 'ar', label: 'Arabic', omt: 'ar' },
```

Comment:
I can not localize the labels in Kibana because the `id` in the `i18n.translate` method [must be a string literal](https://github.com/elastic/kibana/blob/main/src/dev/i18n/README.md#examples-and-restrictions). Should we change each label to its native translation, e.g. `Español`, `Français`, `اَلْعَرَبِيَّةُ` etc.? Maybe also include the `key` in parenthesis? Or just use the `key` as the label? Or are we OK using the English label? What do you think?

Reply:
> Should we change each label to its native translation, e.g. Español, Français, اَلْعَرَبِيَّةُ etc.?

I think that's the best approach, but maybe we should confirm this with our docs team?

This feels out of the scope of the client. Maybe the label is something the UIs should take care of and different solutions can be implemented for Landing Page and Kibana.

What do you think on removing the label from the client and I will prepare the mappings between keys, label in English and each local language for the current selected languages and when we agree on a final solution then extend it to the rest of the OMT supported languages.

Reply:
> I think that's the best approach, but maybe we should confirm this with our docs team? A couple of references suggesting this approach [here](https://usersnap.com/blog/design-language-switch/#:~:text=language%20switcher%20as%20part%20of%20the%20main%20navigation) and [here](https://www.smartling.com/resources/101/language-selector-best-practices/#:~:text=the%20flags%20altogether.-,website%20language%20selector%20best%20practices,-There%20are%20%20some).

Good idea. After reviewing the examples, I think displaying the label in the native translation is best. But I can also get feedback from the docs team.

> This feels out of the scope of the client. Maybe the label is something the UIs should take care of and different solutions can be implemented for Landing Page and Kibana.

> What do you think on removing the label from the client and I will prepare the mappings between keys, label in English and each local language for the current selected languages and when we agree on a final solution then extend it to the rest of the OMT supported languages.

👍  I agree with the downstream apps deciding which languages to support. With Kibana, I think I would only consider supporting [these locales](https://www.elastic.co/guide/en/kibana/8.2/i18n-settings-kb.html) for the MVP. We may add support for more languages in future PRs.

Reply:
Addressed at [38d6caf](https://github.com/elastic/ems-client/pull/105/commits/38d6caf887c789079d305c7777d986f1ad01e77c)